cmd/abigen/main.go

```diff
@@ -122,10 +96,13 @@ func init() {
 }
 
 func abigen(c *cli.Context) error {
-	utils.CheckExclusive(c, abiFlag, jsonFlag, solFlag, vyFlag) // Only one source can be selected.
+	utils.CheckExclusive(c, abiFlag, jsonFlag) // Only one source can be selected.
 	if c.GlobalString(pkgFlag.Name) == "" {
 		utils.Fatalf("No destination package specified (--pkg)")
 	}
+	if c.GlobalString(abiFlag.Name) == "" {
```

Comment:
json can also be a source of abigen, right?

Reply:
oh well spotted!